imap_processing/tests/mag/test_mag_l1a.py

```diff
@@ -61,7 +676,7 @@ def test_process_vector_data():
 
     # 36 bytes
     (primary_vectors, secondary_vectors) = MagL1a.process_vector_data(
-        input_data, total_primary_vectors, total_secondary_vectors
+        input_data, total_primary_vectors, total_secondary_vectors, 0
```

Comment:
Maybe use a constant or enum here
```suggestion
        input_data, total_primary_vectors, total_secondary_vectors, UNCOMPRESSED
```